less/added.less

```diff
@@ -0,0 +1,47 @@
+#accordion {
```

Comment:
great job breaking your less files out.  Makes a big difference for readability, and usability for larger projects.  Great habit to be in.